extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -273,6 +273,30 @@ abstract class RangerSparkExtensionSuite extends KyuubiFunSuite with SparkSessio
       doAs("admin", sql(s"DROP DATABASE IF EXISTS $db"))
     }
   }
+
+  test("show table extended") {
```

Comment:
based on this test, nothing actually changed?

Reply:
In PrivilegesBuilder, users should have SELECT privileges on a table to perform DESCRIBE operations on it, and `show table extended xxx` is similar to performing DESCRIBE operations on batched tables, so when it comes to `show tables extended xxx ` you should have SELECT privileges instead of USE privileges on these tables.

Since the tables in the test case either have SELECT privileges or none at all, nothing changes. I will further improve the test case.

Reply:
is this same to hive？

Reply:
> is this same to hive？

in hive, you can describe a table without any privilege.

Reply:
Looks like we can follow hive's behavior

Reply:
> Looks like we can follow hive's behavior

The permission control of hive is very simply, far less than what is considered in the privilegesbuilder. If we imitate the permission control of hive, the privilegesbuilder will reduce a lot of cases.

Reply:
you can submit PRs accordingly if the hive's behavior is reasonable. For example, DESCRIBE table seems that hive's right to me

Reply:
> you can submit PRs accordingly if the hive's behavior is reasonable. For example, DESCRIBE table seems that hive's right to me

My bad, after I try hive access control in real, I found you need SELECT privilege on a table to describe it. 😹

Reply:
> > you can submit PRs accordingly if the hive's behavior is reasonable. For example, DESCRIBE table seems that hive's right to me
> 
> My bad, after I try hive access control in real, I found you need SELECT privilege on a table to describe it. 😹

At least in hive 2.3 you should have SELECT privilege to describe a table.